elide-core/src/test/java/com/yahoo/elide/core/PersistentResourceTest.java

```diff
@@ -97,8 +101,10 @@ public PersistentResourceTest() {
                 null,
                 new RequestScope(null, null, null, new EntityDictionary(new HashMap<>()), null, MOCK_AUDIT_LOGGER)
         );
-        goodUserScope = new RequestScope(null, null, new User(1), dictionary, null, MOCK_AUDIT_LOGGER);
-        badUserScope = new RequestScope(null, null, new User(-1), dictionary, null, MOCK_AUDIT_LOGGER);
+        goodUserScope = new RequestScope(null, mock(DataStoreTransaction.class, Answers.CALLS_REAL_METHODS),
```

Comment:
does `CALLS_REAL_METHODS` cause the mock to fall back on the actual method if you don't provide a mock value?